bench/irmin-pack/tree.ml

```diff
@@ -20,8 +20,57 @@ type op =
   | Copy of key * key
 [@@deriving yojson]
 
+let pp_inode_config fmt = function
+  | `Entries_2 -> Format.fprintf fmt "[2, 5]"
+  | `Entries_32 -> Format.fprintf fmt "[32, 256]"
+
 module Parse_trace = struct
-  let read_commits commits ncommits path =
+  let is_hex_char = function
+    | '0' .. '9' | 'a' .. 'f' | 'A' .. 'F' -> true
+    | _ -> false
+
+  let is_2char_hex s =
+    if String.length s <> 2 then false
+    else s |> String.to_seq |> List.of_seq |> List.for_all is_hex_char
+
+  let is_30char_hex s =
+    if String.length s <> 30 then false
+    else s |> String.to_seq |> List.of_seq |> List.for_all is_hex_char
+
+  let rec flatten_key_suffix = function
+    | a :: b :: c :: d :: e :: f :: tl
+      when is_2char_hex a
+           && is_2char_hex b
+           && is_2char_hex c
+           && is_2char_hex d
+           && is_2char_hex e
+           && is_30char_hex f ->
+        (a ^ b ^ c ^ d ^ e ^ f) :: flatten_key_suffix tl
+    | hd :: tl -> hd :: flatten_key_suffix tl
+    | [] -> []
+
+  (** This function flattens all the 6 step-long chunks forming 38 byte-long
+      hashes to a single step.
+
+      The paths in tezos:
+      https://www.dailambda.jp/blog/2020-05-11-plebeia/#tezos-path *)
```

Comment:
Let's add the path format (i.e. `[0-9a-f]{2}/[0-9a-f]{2}/[0-9a-f]{2}/[0-9a-f]{2}/[0-9a-f]{2}/[0-9a-f]{30}`) in the comment too.